app/models/user.rb

```diff
@@ -0,0 +1,12 @@
+class User < ApplicationRecord
+  has_secure_password
+  has_many :houses, foreign_key: 'user_id'
+  has_many :reservations, foreign_key: 'user_id'
+  # mount_uploader :avatar, AvatarUploader
```

Comment:
- [ ]  Kindly remove this old piece of commented code, in order to improve the app's cleanliness 💪